scripts/schema/cleaner.py

```diff
@@ -178,12 +182,16 @@ def field_assertions_and_warnings(field):
 SHORT_LIMIT = 120
 
 
-def single_line_short_description(schema_or_field):
+def single_line_short_description(schema_or_field, strict_mode_enabled=True):
     short_length = len(schema_or_field['field_details']['short'])
     if "\n" in schema_or_field['field_details']['short'] or short_length > SHORT_LIMIT:
         msg = "Short descriptions must be single line, and under {} characters (current length: {}).\n".format(
             SHORT_LIMIT, short_length)
         msg += "Offending field or field set: {}\nShort description:\n  {}".format(
             schema_or_field['field_details']['name'],
             schema_or_field['field_details']['short'])
-        raise ValueError(msg)
+        if strict_mode_enabled:
+            raise ValueError(msg)
+        else:
+            message = f"{msg}\n\nThis will cause an exception when running in strict mode."
+            warnings.warn(message)
```

Comment:
I like the idea of dropping the same message as a warning, great idea!

Let's capture that in a reusable function elsewhere. Right now we're only stifling this exception, but #781 will also benefit from only raising in strict mode.